source/patterns/01-molecules/figure/figure~ar-caption.yaml

```diff
@@ -7,8 +7,11 @@ content:
     image:
       src: https://picsum.photos/600/400
       alt: ''
+      class: 'figure__image'
 caption:
   heading:
+    attributes:
+      class: ['figure__heading']
```

Comment:
We'll need to add both these classes in the template rather than from outside.

Reply:
Yeah, it's 'broken' if not included. eLife had to have a mechanism of including them from outside due to the limitations of Mustache.

Reply:
I see. Makes sense.